WooCommerce/Classes/POS/Utils/Color+WooCommercePOS.swift

```diff
@@ -89,23 +98,11 @@ extension Color {
 
     // MARK: - Buttons
 
-    static var posPrimaryButtonBackground: Color {
-        return Color(
-            UIColor(
-                light: .withColorStudio(.wooCommercePurple, shade: .shade50),
-                dark: .withColorStudio(.wooCommercePurple, shade: .shade30)
-            )
-        )
-    }
+    static var posPrimaryButtonBackground: Color = .accent
 
-    static var posTextButtonForeground: Color {
-        return Color(
-            UIColor(
-                light: .withColorStudio(.wooCommercePurple, shade: .shade50),
-                dark: .withColorStudio(.wooCommercePurple, shade: .shade30)
-            )
-        )
-    }
+    static var posSecondaryButtonForeground: Color = .accent
```

Comment:
The `POSSecondaryButtonStyle` was referencing `posPrimaryButtonBackground`. When I first changed `posPrimaryButtonBackground` to use a single color for both themes, I added this property. However, I later modified the primary button background again (see the comment above), so this property isn't strictly necessary now. Still, in my opinion, retaining it could enhance clarity and help decouple the components. Let me know your thoughts.